Dockerfile

```diff
@@ -5,7 +5,7 @@
 #
 FROM eclipse-temurin:11.0.18_10-jre
```

Comment:
This doesn't need to be part of this PR; but I think we should change to an alpine image here, just like I started in https://github.com/B3Partners/brmo/pull/1709